src/components/Contacts/ContactDetails/ContactDonationsTab/PartnershipInfo/EditPartnershipInfoModal/EditPartnershipInfoModal.tsx

```diff
@@ -315,7 +314,7 @@ export const EditPartnershipInfoModal: React.FC<
                       >
                         {phases?.map((phase) => [
                           <ListSubheader key={phase?.id}>
-                            {getLocalizedPhase(t, phase?.id)}
+                            {phase?.name}
                           </ListSubheader>,
                           phase?.contactStatuses.map((status) => (
                             <MenuItem key={status} value={status}>
```

Comment:
This isn't showing correctly. I believe this is the same issue with the other frequency inputs

Reply:
![Screenshot 2024-10-30 at 11 30 50 AM](https://github.com/user-attachments/assets/261d1d61-1d4a-4de5-b3a6-2d653f36fc0f)